src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -779,9 +779,11 @@ def build_span_query(trace_id: str, spans_params: SnubaParams, query_spans: list
     return parents_query
 
 
-def pad_span_id(span):
+def pad_span_id(span: str | None) -> str | None:
     """Snuba might return the span id without leading 0s since they're stored as UInt64
     which means a span like 0011 gets converted to an int, then back so we'll get `11` instead"""
+    if span is None:
+        return span
```

Comment:
Is there a case when padding the span id, we should turn `None` into `0`s?

Reply:
Hmm that would make the typing later a lot neater 🤔 lemme give that a try